package.json

```diff
@@ -23,5 +23,13 @@
     "vanilla-framework": "1.6.x",
     "watch-cli": "^0.2.2",
     "global-nav": "^0.1.3"
+  },
+  "dependencies": {
+    "watch-cli": "^0.2.2",
+    "sass-lint": "^1.10.2",
+    "postcss-cli": "^4.1.0",
+    "node-sass": "^4.5.3",
+    "cssnano": "^3.10.0",
+    "autoprefixer": "^6.3.1"
```

Comment:
This has created some duplicated dependencies between the two dependencies sections, we should merge them.